packages/cli/lib/services/__snapshots__/model.service.unit.test.ts.snap

```diff
@@ -272,7 +272,10 @@ export interface NangoProps {
         };
         messages: unknown[];
     } | undefined;
+    rawSaveOutput?: unknown[] | undefined;
+    rawDeleteOutput?: unknown[] | undefined;
```

Comment:
do we need to store the batch operation output in memory? Can we do the same as the proxy and store them in different files that can be concatenated when replaying the execution?